ci/conform-nvim.nix

```diff
@@ -9,5 +9,5 @@ lib.pipe "${vimPlugins.conform-nvim}/lua/conform/formatters" [
   (builtins.filter (lib.hasSuffix ".lua"))
   (map (lib.removeSuffix ".lua"))
   builtins.toJSON
-  (writeText "conform-formatters")
+  (writeText "conform-formatters.json")
```

Comment:
Should we consider [`writers.writeJSON`](https://github.com/NixOS/nixpkgs/blob/e067934d171a7d69737714fde41c9a9f87b4fa6e/pkgs/build-support/writers/data.nix#L67-L76) instead of `toJSON` + `writeText`?

It's an alias of `(pkgs.formats.json { }).generate`.